packages/fxa-settings/src/models/verification/factory.ts

```diff
@@ -0,0 +1,28 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { useContext } from 'react';
+import { VerificationInfo } from './verification-info';
+import { AppContext } from '../AppContext';
+import { AccountRecoveryKeyInfo } from './account-recovery-key-info';
+
+export function CreateVerificationInfo() {
```

Comment:
These factory.ts modules were added to make model construction more 'react like' in our components. There were some comments in passing that using `new` to create models inside a react component wasn't very 'react like'. After adding these factories, the component code admittedly looks cleaner, and it takes away any ambiguity surrounding how to instantiate these models.

Reply:
Thanks for the explanation.